packages/blocks/src/code-block/utils/consts.ts

```diff
@@ -9,6 +9,7 @@ export const FALLBACK_LANG = 'Plain Text';
  */
 export const PLAIN_TEXT_REGISTRATION = {
   id: FALLBACK_LANG,
+  displayName: FALLBACK_LANG,
```

Comment:
You cannot modify this as `Object literal may only specify known properties, and 'displayName' does not exist in type 'ILanguageRegistration'.` 

What about reverting this commit? @lawvs 

CC @doodlewind